docs/finops-hub/data-processing.md

```diff
@@ -25,13 +34,70 @@ sequenceDiagram
-3. The **msexports_ETL_ingestion** pipeline transforms the data to a standard schema and saves the raw data in parquet format to the **ingestion** container.
+3. The **msexports_ETL_ingestion** pipeline saves exported data in parquet format in the **ingestion** container. [Learn more](#about-ingestion).
 4. Power BI reads cost data from the **ingestion** container.
 
 <br>
 
+## ℹ️ About ingestion
+
+FinOps hubs rely on a specific folder path in the **ingestion** container:
+
+```text
+ingestion/{scope-id}/{month}/focuscost
+```
+
+- `ingestion` is the container where the data pipeline saves data.
+- `{scope-id}` is expected to be the fully-qualified resource ID of the scope the data is from.
+- `{month}` is the year and month of the exported data formatted as `yyyyMM`.
+- `focuscost` is the exported dataset.
+  > Hubs 0.2 only supports FOCUS cost exports. Other export types will be added in a future release.
+
+If you need to 
+
+If you need to use hubs to monitor non-Azure data, convert the data to [FOCUS](../focus/README.md) and drop it into the **ingestion** container. Please note this has not been explicitly tested in the latest release. If you experience any issues, please [create an issue](https://aka.ms/finops/toolkit/ideas).
+
+<br>
+
+## ℹ️ About exports
+
+FinOps hubs leverage Cost Management exports to obtain cost data. Cost Management controls the folder structure for the exported data in the **msexports** container. A typical path looks like:
+
+```text
+{container}/{path}/{date-range}/{export-name}/{export-time}/{guid}/{file}
+```
+
+For FinOps hubs, we expect the following:
+
+```text
+msexports/{scope-id}/{date-range}/{export-name}/{export-time}/{guid}/{file}
+```
+
+- `msexports` is the container specified on the export.
+  > Hubs only monitor the **msexports** container, so this is required.
+- `{scope-id}` is the folder path specified on the export.
+  > Hubs use this to identify which scope the data is coming from. We recommend using the scope ID but any value can be used. Example scope IDs include:
+  > | Scope type      | Example value                                                          |
+  > | --------------- | ---------------------------------------------------------------------- |
+  > | Subscription    | `/subscriptions/###`                                                   |
+  > | Resource group  | `/subscriptions/###/resourceGroups/###`                                |
+  > | Billing account | `/providers/Microsoft.Billing/billingAccounts/###`                     |
+  > | Billing profile | `/providers/Microsoft.Billing/billingAccounts/###/billingProfiles/###` |
+- `{export-name}` is the name of the export.
+  > Hubs ignore this folder.
+- `{date-range}` is the date range data being exported.
+  > Hubs use this to identify the month. Format for this folder is `yyyyMMdd-yyyyMMdd`.
+- `{export-time}` is a timestamp of when the export ran.
```

Comment:
```suggestion
- `{export-time}` is a timestamp of when the export ran. Depending on the API version used, this folder may not be present. As an example, the folder is not included in API version `2023-07-01-preview`.
```

Reply:
@erincon01 That's for calling out this folder. This is a change CM made in this preview version of exports. All previous versions had the export timestamp folder. I wanted to include it because they're talking about bringing it back in the GA API version. Do you think this covers that adequately enough? I could also add `[...]` around the folder if that would help convey its optional. Thoughts?